docs/visual-basic/misc/root-folder-cannot-be-renamed.md

```diff
@@ -11,8 +11,8 @@ An attempt has been made to rename the root directory. This may be a result of a
 -   Check the file path to make sure that it is correct.  
   
 ## See also
- [My.Computer.FileSystem.RenameDirectory]xref:Microsoft.VisualBasic.MyServices.FileSystemProxy.RenameDirectory)  
- [My.Computer.FileSystem.RenameFile]xref:Microsoft.VisualBasic.MyServices.FileSystemProxy.RenameFile)  
+- [My.Computer.FileSystem.RenameDirectory](xref:Microsoft.VisualBasic.MyServices.FileSystemProxy.RenameDirectory)
+- [My.Computer.FileSystem.RenameFile](xref:Microsoft.VisualBasic.MyServices.FileSystemProxy.RenameFile)
```

Comment:
The link is broken; it should be xref:Microsoft.VisualBasic.MyServices.FileSystemProxy.RenameFile%2A